gwt/src/main/java/com/googlecode/gwtphonegap/showcase/client/device/DeviceActivity.ui.xml

```diff
@@ -34,6 +34,22 @@
 						<g:HTML>UUID: &nbsp;</g:HTML>
 						<g:HTML ui:field="uuid"></g:HTML>
 					</g:HorizontalPanel>
+					<g:HorizontalPanel>
+						<g:HTML>UUID: &nbsp;</g:HTML>
```

Comment:
This label and the other new ones should be updated to reflect the new fields. For example, This one should be, **Model** the next one **Manufacturer** and so on rather than repeating the **UUID** label.